### ISSUE TITLE
[Feature]: Update Requirements to include WCAG 2.2

### ISSUE THREAD

**Issue by sandrina-p** [IBMa/equal-access#1881]
### Project

able site

### Description

Hi team! 

I noticed [IBMA Requirements](https://www.ibm.com/able/requirements/requirements) doesn't include all criteria for WCAG 2.2. When do you plan to add it all? (And also add it to the "Filters" by Standard.

---

P.S. I want to express how useful and friendly your site is. I know how hard it is to make WCAG more friendly for beginners and yours is my favorite out there. I often use it as an intro reference to my colleagues when talking about a specific criterion. Even though, some criterions don't have design/code examples, what's there is already highly valuable for beginners. And for that, thank you <3

**Comment by philljenkins** [comment]
@sandrina-p Thank you for the compliments and feedback.

If you select the requirements version for "7.3 Preview" (not the default 7.2), then you will see the WCAG 2.2 filter and the additional requirements listed.

![Screenshot 2024-03-25 064434](https://github.com/IBMa/equal-access/assets/16193609/b54257d7-31f8-4b02-93c7-ec14ca5f521c)

**Comment by philljenkins** [comment]
Or, did I misunderstand your question? Are you looking for the additional **_AAA_** criteria?  The harmonized list of requirements includes a unified list of A and AA Success Criteria only.

**Comment by sandrina-p** [comment]
Yes, exactly @philljenkins, for all the AAA criteria.

Later on, I also realized that your Level 1, 2 and 3 are not a direct match of Level A, AA, AAA. I thought it was until I noticed the missing AAA and then read about [Pace of Completion](https://www.ibm.com/able/toolkit/plan/overview/#pace-of-completion) and realized Level 3 is a mix of A and AA.

A few colleagues of mine who I spoke about this haven't also noticed the difference :x

**Comment by philljenkins** [comment]
We do not include AAA criteria, on purpose,  because they are not always applicable to all content.  
> It is not recommended that Level AAA conformance be required as a general policy for entire sites because it is not possible to satisfy all Level AAA Success Criteria for some content.

The Checker tool does flag "**_Recommendations_**" where possible. So, those users get suggestions that are specific to their situation and code without overwhelming them with "too much" guidance in the Toolkit.  

All of the user research we've done has shown that teams appreciate the [Pace of Completion](https://www.ibm.com/able/toolkit/plan/overview/#pace-of-completion) approach, especially for new projects and new acquisitions to IBM. 

Of course, our core IBM Accessibility team and many focals in the product teams are Subject Matter Experts (SMEs) and use the AAA criteria. Some of us have been and still are on the W3C working groups too. 

Our experience has also shown that some non-SMEs can get "distracted" and "invent requirements" that do not contribute to good accessibility. I think some overlay features are an example of that where the user already has the AT and preferences that they need & want, while the overlay is redundant and confusing. In my opinion, some AAA criteria (not all) are redundant with AT features, and that is one of the reasons that the working groups reached consensus on AAA.
